src/main.py

```diff
@@ -61,7 +61,7 @@ def frontend_communication():
         "notes": [
             {
                 "symbol": note.symbol,
-                "rhythmic_value": note.rhythmic_value
+                "duration": note.duration
```

Comment:
we need to return rhythmic value (convert from duration to rhythmic value here)

Reply:
Done